egs2/voxceleb/spk1/local/data.sh

```diff
@@ -151,7 +151,8 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
     for x in music noise speech; do
         find ${data_dir_prefix}/musan/${x} -iname "*.wav" > ${data_dir_prefix}/musan_${x}.scp
     done
-    find ${data_dir_prefix}/RIRS_NOISES/simulated_rirs -iname "*.wav" > ${data_dir_prefix}/rirs.scp
+    find ${data_dir_prefix}/RIRS_NOISES/simulated_rirs/mediumroom -iname "*.wav" > ${data_dir_prefix}/rirs.scp
```

Comment:
I recommend you leave the comments that you only use the medium and small rooms. I remember that Kaldi recipes often skip the large room (same as your setup), and they clearly documented this usage.

Reply:
sure. I will add a comment on this. Yes, it follows kaldi's setup.